packages/api/tsconfig.json

```diff
@@ -28,5 +28,6 @@
     }
   },
   "include": ["src/**/*.ts", "test/**/*.ts", "src/**/*.json"],
+  "exclude": ["src/scripts/*.ts"],
```

Comment:
This exclude is not necessary - the build errors I'm getting from those files look accurate and should be fixed.

Reply:
👍